corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -292,20 +296,20 @@ public int loadTable(String namespace, String tablename, int numItems, int batch
     public long listenOnTable(String namespace, String tableName, int stopAfter) {
         verifyNamespaceAndTablename(namespace, tableName);
         CorfuStoreShim store = new CorfuStoreShim(runtime);
-        final Table<TableName, TableName, ManagedMetadata> table;
+        final Table<ExampleTableName, ExampleTableName, ManagedMetadata> table;
         try {
             TableOptions.TableOptionsBuilder<Object, Object> optionsBuilder = TableOptions.builder();
             if (diskPath != null) {
                 optionsBuilder.persistentDataPath(Paths.get(diskPath));
             }
             table = store.openTable(
                     namespace, tableName,
-                    TableName.class,
-                    TableName.class,
+                    ExampleTableName.class,
+                    ExampleTableName.class,
                     ManagedMetadata.class,
                     optionsBuilder.build());
         } catch (Exception ex) {
-            log.error("Unable to open table "+namespace+"$"+tableName);
+            log.error("Unable to open table " + namespace + "$" + tableName);
```

Comment:
should we also throw an exception here?  Otherwise it will only log in the file.

Reply:
done.